frontend/packages/core/src/tab.tsx

```diff
@@ -48,14 +48,15 @@ const StyledTabs = styled(TabList)({
   },
 });
 
-export interface TabProps extends Pick<MuiTabProps, "label" | "selected" | "value" | "onClick"> {
+export interface TabProps extends Pick<MuiTabProps, "label" | "value" | "onClick"> {
   children?: React.ReactNode;
   startAdornment?: React.ReactNode;
+  selected?: boolean;
```

Comment:
should we pass in `selected` like we do `startAdornment`?

Reply:
As in pull it onto the tab component and apply it? We could, but it's already applied by spreading the props. Unless you're referring to something else.

This was only added so that it would setup the `mui-selected` class.

Reply:
ahh i see, nvm!